cmd/component.go

```diff
@@ -212,7 +217,12 @@ var componentSetCmd = &cobra.Command{
 		return nil
 	}, Run: func(cmd *cobra.Command, args []string) {
 		client := getOcClient()
-		err := component.SetCurrent(client, args[0])
+		currentApplication, err := application.GetCurrentOrDefault(client)
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+		err = component.SetCurrent(client, args[0], currentApplication)
```

Comment:
We will probably need to rebase this after #279 is merged in, but since these are two functions which are copied / pasted. Should we put this under a function instead?

Perhaps a `getAndSetApplication()` function?

Reply:
> Perhaps a getAndSetApplication() function?

It is getting a current application but setting current component. 
It is better to decouple this logic.